cms/envs/devstack.py

```diff
@@ -209,6 +209,10 @@ def should_show_debug_toolbar(request):
 # Dummy secret key for dev
 SECRET_KEY = '85920908f28904ed733fe576320db18cabd7b6cd'
 
+FEATURES.update({
```

Comment:
@elvinquero I would put this here: https://github.com/Pearson-Advance/edx-platform/blob/3792f3d740d76b245f7ff05492ede2ce9a637129/cms/envs/common.py#L164
and keep in mind that you will need to set this variable in ansible in order to retrieve the variable in production, take a look at this: https://github.com/Pearson-Advance/edx-platform/blob/3792f3d740d76b245f7ff05492ede2ce9a637129/cms/envs/production.py#L273